packages/advanced-logic/test/utils/payment-network/erc777/stream-create-data-generator.ts

```diff
@@ -95,6 +109,29 @@ export const extensionFullState = {
     version: '0.1.0',
   },
 };
+export const extensionFullStateSubsequent = {
```

Comment:
Please type these objects, it helps a lot navigating and understanding extension types.